test/test-fs.c

```diff
@@ -1257,6 +1257,8 @@ TEST_IMPL(fs_fstat) {
          s->st_birthtim.tv_sec == t.st_ctim.tv_sec);
   ASSERT(s->st_birthtim.tv_nsec == 0 ||
          s->st_birthtim.tv_nsec == t.st_ctim.tv_nsec);
+  ASSERT(s->st_flags == 0);
```

Comment:
perhaps it's worthwhile to add a call at the top to poison the memory? `memset(s, 0xaa, sizeof(*s))`